libs/database/src/lib/entities/nursery-report.entity.ts

```diff
@@ -24,7 +24,8 @@ import { Subquery } from "../util/subquery.builder";
 @Scopes(() => ({
   incomplete: { where: { status: { [Op.notIn]: COMPLETE_REPORT_STATUSES } } },
   nurseries: (ids: number[] | Literal) => ({ where: { nurseryId: { [Op.in]: ids } } }),
-  approved: { where: { status: { [Op.in]: NurseryReport.APPROVED_STATUSES } } }
+  approved: { where: { status: { [Op.in]: NurseryReport.APPROVED_STATUSES } } },
+  reportsTask: (taskId: number) => ({ where: { taskId: taskId } })
```

Comment:
Same feedback here as on site report - I think this would be better named simply `task`